logic.py

```diff
@@ -1,6 +1,6 @@
 ## conditons when we have list of possible values 
```

Comment:
GPT summary of [ff48b2](https://github.com/SampatSuthar/ml-demo/blob/2ce9a211365f18cfb46299b15c0e532c3e8af178/logic.py#ff48b223391a0063edf03dca03d22da628bb8158) - [786a11](https://github.com/SampatSuthar/ml-demo/blob/beb700f9103db16237e7104e4f08463da638e0b3/logic.py#786a110b31b59ea64dca97da186af1d49882f39c):
Error: couldn't generate summary